source/crud/read-operations/project.txt

```diff
@@ -0,0 +1,94 @@
+==============================
+Specify Which Fields to Return
+==============================
+
+.. default-domain:: mongodb
+
+Use a projection to control which fields appear in the documents
+returned by read operations. You can accomplish this in two
+ways:
```

Comment:
Can we add a bit of motivation?

- You may only want some fields from documents that you query
- Useful for matching your application interfaces
- Useful for minimizing network bandwidth